app/models/user.rb

```diff
@@ -312,6 +312,9 @@ def like_for(target)
   ######### Data export ##################
   mount_uploader :export, ExportedUser
 
+  ######### Photo export ##################
```

Comment:
It felt better to see the mount_uploader: (mixins?) to be nearby. Then the processing parts follow in order.

Reply:
Before the `mount_uploader` were grouped together with the respective processing for them. The `User` model has a lot of comment header grouping together different parts and this mixes this up because now the `Photo export` header is above the processing for the `Data export`. If you want to make this one big group (and both the `mount_uploader` together at the top) then you would need to remove the second header and create a header for both data and photos together.

Reply:
Makes sense.I'll change this back.